### ISSUE TITLE
Error copying ionic-angular/fonts to www v0.46

### ISSUE THREAD

**Issue by napcat** [driftyco/ionic-app-scripts#475]
#### Short description of the problem:
When upgrading to rc3 and using the app-scripts 0.46 there's an error during the build.
If the 0.45 version is used there's no errors.

#### What behavior are you expecting?
Build with no errors with app scripts 0.46.

**Steps to reproduce:**
1. Migrate from rc2 to rc3.
2. Update version of ionic-app-scripts from 0.45 to 0.46 and run npm install
3. run ionic serve

**Which @ionic/app-scripts version are you using?**
0.46

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

The fonts files seems to be copied to de directorie in the www, but theres this error:

```
[10:06:19]  ionic-app-scripts 0.0.46
[10:06:19]  watch started ...
[10:06:19]  build dev started ...
[10:06:19]  clean started ...
[10:06:19]  clean finished in less than 1 ms
[10:06:19]  copy started ...
[10:06:19]  transpile started ...
[10:06:23]  transpile finished in 4.01 s
[10:06:23]  webpack started ...
[10:06:23]  copy: Error copying "D:\AppSmartIonic\AppSmartIonic-rc3\node_modules\ionic-angular\fonts" to
            "D:\AppSmartIonic\AppSmartIonic-rc3\www\assets\fonts"
[10:06:23]  copy finished in 4.25 s
[10:06:30]  webpack finished in 7.18 s
[10:06:30]  sass started ...
[10:06:32]  sass finished in 1.37 s
[10:06:32]  build dev finished in 12.58 s
[10:06:32]  watch ready in 12.78 s
[10:06:32]  dev server running: http://localhost:8100/
```

**Comment by fiznool** [comment]
I believe this is a duplicate of https://github.com/driftyco/ionic-app-scripts/issues/448 ?

**Comment by kodeine** [comment]
i get the same error.

**Comment by ozexpert** [comment]
what is the fix for this? getting same error. manually copying for now.

**Comment by snowyu** [comment]
confirmed.

**Comment by fxlemire** [comment]
@ozexpert I believe it is fixed in `0.0.47`. So simply run `npm install @ionic/app-scripts@latest --save-dev`.

**Comment by snowyu** [comment]
yes, fixed in 0.0.47.

**Comment by danbucholtz** [comment]
This is fixed now.

Thanks,
Dan
